### DISCUSSION TITLE
I can't use chatgpt

### DISCUSSION THREAD

**Discussion by playdasegunda** [danny-avila/LibreChat#41] [Troubleshooting] [answered]
I don't know why but when I try to chat with chatgpt-clone I get error, this is the error:

![1](https://user-images.githubusercontent.com/97372669/224477758-32bf5816-f9d4-4053-ab27-09888ba2ade1.jpg)
![2](https://user-images.githubusercontent.com/97372669/224477760-5d086c66-8c28-497c-9750-437745d202ff.jpg)

my .env file:
![3](https://user-images.githubusercontent.com/97372669/224477773-5102137b-9e06-4828-b57f-f1b1246ab6f6.jpg)

I can't use any options: ChatGPT, CustomGPT$..

What could be causing this problem?

**Comment by danny-avila** [comment, marked as answer]
Looks like you are setting `CHATGPT_TOKEN ` to your API key instead of your access token for the free version. Read the instructions here on how to get your user token for the free version. https://github.com/danny-avila/chatgpt-clone#access-tokens

Set your API key ( what you currently have under `CHATGPT_TOKEN`) to `OPENAI_key`
